Sensus.Android/Sensus.Android.csproj

```diff
@@ -417,9 +425,7 @@
       <Link>Resources\drawable\share.png</Link>
     </AndroidResource>
   </ItemGroup>
-  <ItemGroup>
-    <Folder Include="Assets\" />
```

Comment:
Yes, and it's still there in the IDE. I opened the solution on both OS X and Windows to confirm. I also opened the .csproj file in a text editor to make sure I actually have this change (in case I was accidentally in the wrong branch or something).

I was curious, so I did a little testing. It looks like, in VS 2015 at least, if a folder is added to a project without any contents then the <Folder> tag is added to keep track of the folder. However, once an item is added to the folder VS removes the pure <Folder> tag from the csproj file and simply tracks the item in that folder. It looks like VS and XS are both smart enough to load all folder hierarchies for their items so tracking Folders explicitly isn't necessary.